README.md

```diff
@@ -1,3 +1,16 @@
+# Adobe User Sync Tool
```

Comment:
"Adobe User Sync Tool" violates brand name guidelines and can't be used.  You have to say "User Sync Tool from Adobe"